libsel4bench/arch_include/arm/cpu/cortex-a35/sel4bench/cpu/events.h

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright 2017, Data61, CSIRO (ABN 41 687 119 230)
```

Comment:
Don't you claim the copyright?

Reply:
I do not. The file is a direct copy from existing data61 copyrighted code.